<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ddaWidget</class>
 <widget class="QWidget" name="ddaWidget">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>930</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Differential Expression Analysis</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1084</width>
        <height>904</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QWidget" name="plotWidget" native="true">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>400</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>550</width>
               <height>400</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Selection B</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Selection A </string>
               </property>
              </widget>
             </item>
             <item row="11" column="0" colspan="2">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Number of Genes only present in Selection A</string>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Correlation between distributions</string>
               </property>
              </widget>
             </item>
             <item row="8" column="2">
              <widget class="QLineEdit" name="stdDevSelectionB">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Number of Genes in Selection B</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0" colspan="2">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Standard Dev of distribution in Selection A</string>
               </property>
              </widget>
             </item>
             <item row="11" column="2">
              <widget class="QLineEdit" name="genesOnlyA">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLineEdit" name="numGenesSelectionA">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QLineEdit" name="numGenesSelectionB">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="12" column="0" colspan="2">
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>Number of Genes only present in Selection B</string>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QLineEdit" name="meanSelectionA">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="10" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Number of Overlapping Genes</string>
               </property>
              </widget>
             </item>
             <item row="7" column="2">
              <widget class="QLineEdit" name="stdDevSelectionA">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1" colspan="2">
              <widget class="QLineEdit" name="selectionB">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Mean of distribution in Selection B</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QLineEdit" name="selectionA">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Mean of distribution in Selection A</string>
               </property>
              </widget>
             </item>
             <item row="12" column="2">
              <widget class="QLineEdit" name="genesOnlyB">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="9" column="2">
              <widget class="QLineEdit" name="correlation">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Statistics Report</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0" colspan="2">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Standard Dev of distribution in Selection B</string>
               </property>
              </widget>
             </item>
             <item row="10" column="2">
              <widget class="QLineEdit" name="overlappingGenes">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="6" column="2">
              <widget class="QLineEdit" name="meanSelectionB">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Number of Genes in Selection A</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="toolTip">
            <string/>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Save the DEA to a file</string>
           </property>
           <property name="statusTip">
            <string>Save the DEA to a file</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
           <property name="icon">
            <iconset resource="../../../st_client-build/application.qrc">
             <normaloff>:/images/filesave.png</normaloff>:/images/filesave.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Close the DEA</string>
           </property>
           <property name="statusTip">
            <string>Close the DEA</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="icon">
            <iconset resource="../../../st_client-build/application.qrc">
             <normaloff>:/images/abort.png</normaloff>:/images/abort.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../st_client-build/application.qrc"/>
 </resources>
 <connections/>
</ui>
